test/DefaultCluster.Tests/BasicActivationTests.cs

```diff
@@ -11,11 +11,12 @@
 
 #pragma warning disable 618
 
-namespace UnitTests.General
+namespace DefaultCluster.Tests.General
 {
+    [TestCategory("BVT"), TestCategory("Functional")]
```

Comment:
were these tests converted to BVTs on purpose? Because they weren't before.
BTW, we should not mark the whole class as BVT or Functional, and have each test be marked with it (as there might be a class that have a few BVTs and some that arent't, and it would be easy to miss when adding new test methods). Nevertheless,  big categories such as "ActivateDeactivate" is fine to add them at class level.